dotcom-rendering/src/components/LoopVideo.importable.tsx

```diff
@@ -60,13 +61,24 @@ const dispatchOphanAttentionEvent = (
 	document.dispatchEvent(event);
 };
 
+const getOptimisedPosterImage = (mainImage: string): string => {
+	const resolution = window.devicePixelRatio >= 2 ? 'high' : 'low';
+
+	return generateImageURL({
+		mainImage,
+		imageWidth: 940, // The widest a looping video can be: Flexible special, giga-boosted
```

Comment:
This isn't ideal. We'd like to serve a smaller image on smaller screens. Unfortunately, multiple sources are not supported for the poster image of a video.

We could introduce some logic to optimise this, but would require a bit of thought to how it could be maintained if cards/containers change. One for a follow-up PR